Build/Module.xml

```diff
@@ -12,5 +12,9 @@
     <Package Uri="https-git://github.com/OutOfOrder/MonoKickstart.git" Folder="ThirdParty/Kickstart" GitRef="master" />
```

Comment:
I thought this might cause the Xwt,Gtk,dll not being in the right place. I had the same issue when trying to add NUnit as a NuGet package with Protobuild and setting GenerateNuGetRepositories to true fixed it.